BubblesKotlin/app/src/main/java/com/example/android/bubbles/ui/chat/ChatFragment.kt

```diff
@@ -80,38 +74,16 @@ class ChatFragment : Fragment() {
         return inflater.inflate(R.layout.chat_fragment, container, false)
     }
 
-    private val startPostponedTransitionOnEnd = object : RequestListener<Drawable> {
-        override fun onLoadFailed(
-            e: GlideException?,
-            model: Any?,
-            target: Target<Drawable>?,
-            isFirstResource: Boolean
-        ): Boolean {
-            startPostponedEnterTransition()
-            return false
-        }
-
-        override fun onResourceReady(
-            resource: Drawable?,
-            model: Any?,
-            target: Target<Drawable>?,
-            dataSource: DataSource?,
-            isFirstResource: Boolean
-        ): Boolean {
-            startPostponedEnterTransition()
-            return false
-        }
-    }
-
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         val id = arguments?.getLong(ARG_ID)
         if (id == null) {
-            fragmentManager?.popBackStack()
+            if (isAdded) {
```

Comment:
This feels more like an exceptional state, so throwing an exception is better?

Reply:
Done. Removed `if (isAdded)` so this will be an exception if it ever happens.